addons/account/data/account_data.xml

```diff
@@ -91,6 +91,15 @@
                 Command.create({'value': 'balance', 'value_amount': 0.0, 'days': 60})]"/>
         </record>
 
+        <record id="account_payment_term_30days_early_discount" model="account.payment.term">
+            <field name="name">2/7 Net 30</field>
```

Comment:
@odoostan Is that clear for the user with the 2/7 ?

Reply:
Maybe there is also a task you could refer to?

Reply:
It's not clear to me, but it is to @chklop. 
No task was provided in the making of this PR.

Reply:
@chklop Users understand that?

Reply:
Hello @jco-odoo the notation is kind of known (you can google it easily). Also, that name is only relevant in the interface were the user uses it as a reference, so short is better.

If you think that's too much, an alternative could be: "30 days, Discount 2% under 7 days"

Reply:
For accountants, nice, for developers, it is weird.  If we would do -2% if < 7 days it would be too developer probably.